tests/Struct/FindologicProductTest.php

```diff
@@ -390,13 +389,23 @@ public function testProduct(): void
             $config
         );
 
+        $productKeyword = new Keyword('FINDOLOGIC001');
```

Comment:
Where are you assigning the keyword to the product? We have already established that the product number and vendor name is returned in the keywords by default, so we cannot test it against the product number, because the test is not checking if our configured keyword is getting returned or not. So instead of checking against default values, assign a random keyword to the product and make sure that is returned. 
Theoretically this is fine too since we are getting the keyword back, but I guess @TheKeymaster can have a better opinion on that.